### PR TITLE
fix moesif data collector

### PR DESCRIPTION
**Pull request by tharindu1st**
fix moesif data collector

### CODE REVIEW

File: component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/retriever/ConfigBaseMoesifKeyRetriever.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.wso2.am.analytics.publisher.retriever;
```
Comment (tishan89) [inline component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/retriever/ConfigBaseMoesifKeyRetriever.java:1]: license

File: component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/GenericInputValidator.java [hunk 2]
```diff
@@ -86,6 +87,8 @@ public class GenericInputValidator {
                     new AbstractMap.SimpleImmutableEntry<>(APPLICATION_ID, String.class),
                     new AbstractMap.SimpleImmutableEntry<>(APPLICATION_NAME, String.class),
                     new AbstractMap.SimpleImmutableEntry<>(APPLICATION_OWNER, String.class),
+                    new AbstractMap.SimpleImmutableEntry<>(ORGANIZATION_ID, String.class),
```
Comment (tishan89) [inline component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/GenericInputValidator.java:90]: Please make sure there new attributes are set by MI to the builder. Else publishing will fail for even the Choreo flow

**Review by tishan89 — CHANGES_REQUESTED** [review]

File: component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/GenericInputValidator.java [hunk 3]
```diff
@@ -67,6 +67,7 @@
  * inputs will be validated.
  */
 public class GenericInputValidator {
+
     private static final org.wso2.am.analytics.publisher.reporter.GenericInputValidator INSTANCE =
             new org.wso2.am.analytics.publisher.reporter.GenericInputValidator();
     private static final Map<String, Class> defaultResponseEventSchema = Stream.of(
```
Comment (ruks) [inline component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/GenericInputValidator.java:73]: Are we adding these new attributes only for moseif? or same published for choreo as well?

**Comment by tharindu1st** [comment]
Closing this pr with https://github.com/wso2/apk/pull/1834

**Pull request closed without merging**